xwiki-platform-core/xwiki-platform-tag/xwiki-platform-tag-ui/src/main/resources/XWiki/TagCloud.xml

```diff
@@ -743,4 +757,82 @@ $xwiki.ssx.use('XWiki.TagCloud')##
       <type/>
     </property>
   </object>
+  <object>
+    <name>XWiki.TagCloud</name>
```

Comment:
This parameter is missing translations:
- `rendering.macro.tagcloud.parameter.limit.name`
- `rendering.macro.tagcloud.parameter.limit.description`